tsconfig.build.json

```diff
@@ -4,6 +4,7 @@
   "compilerOptions": {
     "declaration": true,
     "declarationMap": true,
+    "noEmit": false,
```

Comment:
@lenkan 
Regarding [this](https://github.com/WebOfTrust/polaris-web/pull/26/files#r1757144479) I have explicitly added noEmit:false in tsconfig.build. That way it would generate dist folder